server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQTest.java

```diff
@@ -63,6 +65,24 @@ public void publishedEventWithoutSubscriberShouldNotBeLost() throws Exception {
         }
     }
 
+    @Test
+    public void demonstrateDurability() throws Exception {
+        try (Connection connection = connectionFactory.newConnection();
+                Channel channel = connection.createChannel();) {
+            String exchangeName = "exchangeName";
+            String routingKey = "routingKey";
+            String queueName = createQueue(channel, exchangeName, routingKey);
+
+            publishAMessage(channel, exchangeName, routingKey);
+
+            rabbitMQ.restart();
+
+            Thread.sleep(ONE_MINUTE);
```

Comment:
we should use WaitStrategy here

Reply:
You mean the one from **testcontainers**? It won't work here as we bypass it.

We need polling with awaitability.